src/HtmlReader/HtmlReaderContent.tsx

```diff
@@ -2,13 +2,11 @@ import * as React from 'react';
 
 const HtmlReaderContent = () => {
   return (
-    <div style={{ height: '100%', overflow: 'hidden', width: '100%' }}>
-      <div id="D2Reader-Container">
-        <main style={{ height: '100vh' }} tabIndex={-1} id="iframe-wrapper">
-          <div id="reader-loading" className="loading"></div>
-          <div id="reader-error" className="error"></div>
-        </main>
-      </div>
+    <div id="D2Reader-Container">
```

Comment:
Is this being added because there are styles coming from the r2 library?

Reply:
This is necessary because the D2reader looks specifically for the object with this id to inject the reader

https://github.com/d-i-t-a/R2D2BC/blob/8543345bcd91f1ae740b7910fa9ca8bed7717aca/src/index.ts#L568
(When `iframeNavigator.start` is called, the rest of the reader divs get created underneath it)

Reply:
Got it, thanks!